auth/token-managers/cp4d-token-manager.ts

```diff
@@ -16,6 +16,7 @@
 
 import extend from 'extend';
 import { validateInput } from '../utils/helpers';
+import { buildUserAgent } from '../../lib/privateHelper';
```

Comment:
Convention is to use kebab-case file names.

Also, this is picky, but if we might add other private helpers to that file perhaps it should be plural? If not, maybe it should just be called `build-user-agent`.

```suggestion
import { buildUserAgent } from '../../lib/private-helpers';
```

Reply:
fixed